src/collections/landscape/non-functional.js

```diff
@@ -200,19 +199,6 @@ export const non_functional=[{
   timeline_order: 14,
   icon: SofaMesh
 },
-{
-  name: "Octarine",
-  opensource: "No",
-  link: "https://www.octarinesec.com/",
```

Comment:
This non-functional is only part of the landscape update that would be needed (functional update needed, too).

Let's leave Octarine in the landscape, but list as Project Shutdown in the first column of the Functional table.

Reply:
Done